### ISSUE TITLE
[time-input & date-time-picker] Label has Insufficient color contract in Mobile Mode

### ISSUE THREAD

**Issue by emilyrohrbough** [cerner/terra-framework#744]
# Bug Report

## Description
Insufficient color contract for time-input && date-time-picker label in tiny viewport for mobile rendering

## Steps to Reproduce

1. [Remove ignore rules in test ](https://github.com/cerner/terra-framework/blob/master/packages/terra-date-time-picker/tests/wdio/date-time-picker-mobile-spec.js#L2-L4)
2. Run tests & see error

<details>
<summary>Error output</summary>

```
running chrome
AssertionError: expected no accessibility violations but got:
	{
  "description": "Ensures the contrast between foreground and background colors meets WCAG 2 AA contrast ratio thresholds",
  "help": "Elements must have sufficient color contrast",
  "helpUrl": "https://dequeuniversity.com/rules/axe/3.2/color-contrast?application=axeAPI",
  "id": "color-contrast",
  "impact": "serious",
  "nodes": [
    {
      "all": [],
      "any": [
        {
          "data": {
            "bgColor": "#ffffff",
            "contrastRatio": 3.97,
            "expectedContrastRatio": "4.5:1",
            "fgColor": "#808080",
            "fontSize": "9.0pt",
            "fontWeight": "normal"
          },
          "id": "color-contrast",
          "impact": "serious",
          "message": "Element has insufficient color contrast of 3.97 (foreground color: #808080, background color: #ffffff, font size: 9.0pt, font weight: normal). Expected contrast ratio of 4.5:1",
          "relatedNodes": [
            {
              "html": "<html lang=\"en\" dir=\"ltr\">",
              "target": [
                "html"
              ]
            }
          ]
        }
      ],
      "failureSummary": "Fix any of the following:\n  Element has insufficient color contrast of 3.97 (foreground color: #808080, background color: #ffffff, font size: 9.0pt, font weight: normal). Expected contrast ratio of 4.5:1",
      "html": "<label for=\"terra-time-hour-input\" class=\"TimeInput-module__mobile-input-label___2W6me\">Hours</label>",
      "impact": "serious",
      "none": [],
      "target": [
        "#datetimeEmpty > .DateTimePicker-module__time-facade___RbC0N > .TimeInput-module__mobile-time-picker___3RdnY > .TimeInput-module__time-input-group___9V3oL:nth-child(2) > label[for=\"terra-time-hour-input\"]"
      ]
    },
    {
      "all": [],
      "any": [
        {
          "data": {
            "bgColor": "#ffffff",
            "contrastRatio": 3.97,
            "expectedContrastRatio": "4.5:1",
            "fgColor": "#808080",
            "fontSize": "9.0pt",
            "fontWeight": "normal"
          },
          "id": "color-contrast",
          "impact": "serious",
          "message": "Element has insufficient color contrast of 3.97 (foreground color: #808080, background color: #ffffff, font size: 9.0pt, font weight: normal). Expected contrast ratio of 4.5:1",
          "relatedNodes": [
            {
              "html": "<html lang=\"en\" dir=\"ltr\">",
              "target": [
                "html"
              ]
            }
          ]
        }
      ],
      "failureSummary": "Fix any of the following:\n  Element has insufficient color contrast of 3.97 (foreground color: #808080, background color: #ffffff, font size: 9.0pt, font weight: normal). Expected contrast ratio of 4.5:1",
      "html": "<label for=\"terra-time-minute-input\" class=\"TimeInput-module__mobile-input-label___2W6me\">Minutes</label>",
      "impact": "serious",
      "none": [],
      "target": [
        "#datetimeEmpty > .DateTimePicker-module__time-facade___RbC0N > .TimeInput-module__mobile-time-picker___3RdnY > .TimeInput-module__time-input-group___9V3oL:nth-child(4) > label[for=\"terra-time-minute-input\"]"
      ]
    },
    {
      "all": [],
      "any": [
        {
          "data": {
            "bgColor": "#ffffff",
            "contrastRatio": 3.97,
            "expectedContrastRatio": "4.5:1",
            "fgColor": "#808080",
            "fontSize": "9.0pt",
            "fontWeight": "normal"
          },
          "id": "color-contrast",
          "impact": "serious",
          "message": "Element has insufficient color contrast of 3.97 (foreground color: #808080, background color: #ffffff, font size: 9.0pt, font weight: normal). Expected contrast ratio of 4.5:1",
          "relatedNodes": [
            {
              "html": "<html lang=\"en\" dir=\"ltr\">",
              "target": [
                "html"
              ]
            }
          ]
        }
      ],
      "failureSummary": "Fix any of the following:\n  Element has insufficient color contrast of 3.97 (foreground color: #808080, background color: #ffffff, font size: 9.0pt, font weight: normal). Expected contrast ratio of 4.5:1",
      "html": "<label for=\"terra-time-hour-input\" class=\"TimeInput-module__mobile-input-label___2W6me\">Hours</label>",
      "impact": "serious",
      "none": [],
      "target": [
        "#datetimeValueProvided > .DateTimePicker-module__time-facade___RbC0N > .TimeInput-module__mobile-time-picker___3RdnY > .TimeInput-module__time-input-group___9V3oL:nth-child(2) > label[for=\"terra-time-hour-input\"]"
      ]
    },
    {
      "all": [],
      "any": [
        {
          "data": {
            "bgColor": "#ffffff",
            "contrastRatio": 3.97,
            "expectedContrastRatio": "4.5:1",
            "fgColor": "#808080",
            "fontSize": "9.0pt",
            "fontWeight": "normal"
          },
          "id": "color-contrast",
          "impact": "serious",
          "message": "Element has insufficient color contrast of 3.97 (foreground color: #808080, background color: #ffffff, font size: 9.0pt, font weight: normal). Expected contrast ratio of 4.5:1",
          "relatedNodes": [
            {
              "html": "<html lang=\"en\" dir=\"ltr\">",
              "target": [
                "html"
              ]
            }
          ]
        }
      ],
      "failureSummary": "Fix any of the following:\n  Element has insufficient color contrast of 3.97 (foreground color: #808080, background color: #ffffff, font size: 9.0pt, font weight: normal). Expected contrast ratio of 4.5:1",
      "html": "<label for=\"terra-time-minute-input\" class=\"TimeInput-module__mobile-input-label___2W6me\">Minutes</label>",
      "impact": "serious",
      "none": [],
      "target": [
        "#datetimeValueProvided > .DateTimePicker-module__time-facade___RbC0N > .TimeInput-module__mobile-time-picker___3RdnY > .TimeInput-module__time-input-group___9V3oL:nth-child(4) > label[for=\"terra-time-minute-input\"]"
      ]
    },
    {
      "all": [],
      "any": [
        {
          "data": {
            "bgColor": "#ffffff",
            "contrastRatio": 3.97,
            "expectedContrastRatio": "4.5:1",
            "fgColor": "#808080",
            "fontSize": "9.0pt",
            "fontWeight": "normal"
          },
          "id": "color-contrast",
          "impact": "serious",
          "message": "Element has insufficient color contrast of 3.97 (foreground color: #808080, background color: #ffffff, font size: 9.0pt, font weight: normal). Expected contrast ratio of 4.5:1",
          "relatedNodes": [
            {
              "html": "<html lang=\"en\" dir=\"ltr\">",
              "target": [
                "html"
              ]
            }
          ]
        }
      ],
      "failureSummary": "Fix any of the following:\n  Element has insufficient color contrast of 3.97 (foreground color: #808080, background color: #ffffff, font size: 9.0pt, font weight: normal). Expected contrast ratio of 4.5:1",
      "html": "<label for=\"terra-time-hour-input\" class=\"TimeInput-module__mobile-input-label___2W6me\">Hours</label>",
      "impact": "serious",
      "none": [],
      "target": [
        "#disabledDatetimeValueProvided > .DateTimePicker-module__time-facade___RbC0N > .TimeInput-module__mobile-time-picker___3RdnY > .TimeInput-module__time-input-group___9V3oL:nth-child(2) > label[for=\"terra-time-hour-input\"]"
      ]
    },
    {
      "all": [],
      "any": [
        {
          "data": {
            "bgColor": "#ffffff",
            "contrastRatio": 3.97,
            "expectedContrastRatio": "4.5:1",
            "fgColor": "#808080",
            "fontSize": "9.0pt",
            "fontWeight": "normal"
          },
          "id": "color-contrast",
          "impact": "serious",
          "message": "Element has insufficient color contrast of 3.97 (foreground color: #808080, background color: #ffffff, font size: 9.0pt, font weight: normal). Expected contrast ratio of 4.5:1",
          "relatedNodes": [
            {
              "html": "<html lang=\"en\" dir=\"ltr\">",
              "target": [
                "html"
              ]
            }
          ]
        }
      ],
      "failureSummary": "Fix any of the following:\n  Element has insufficient color contrast of 3.97 (foreground color: #808080, background color: #ffffff, font size: 9.0pt, font weight: normal). Expected contrast ratio of 4.5:1",
      "html": "<label for=\"terra-time-minute-input\" class=\"TimeInput-module__mobile-input-label___2W6me\">Minutes</label>",
      "impact": "serious",
      "none": [],
      "target": [
        "#disabledDatetimeValueProvided > .DateTimePicker-module__time-facade___RbC0N > .TimeInput-module__mobile-time-picker___3RdnY > .TimeInput-module__time-input-group___9V3oL:nth-child(4) > label[for=\"terra-time-minute-input\"]"
      ]
    }
  ],
  "tags": [
    "cat.color",
    "wcag2aa",
    "wcag143"
  ]
}
    at Object.runAccessibilityTest (/Users/er047227/terra-framework/node_modules/terra-toolkit/lib/wdio/services/TerraCommands/accessibility.js:23:35)
    at Context.<anonymous> (/Users/er047227/terra-framework/node_modules/terra-toolkit/lib/wdio/services/TerraCommands/validate-element.js:77:28)
    at Promise (<anonymous>)
    at F (/Users/er047227/terra-framework/node_modules/babel-runtime/node_modules/core-js/library/modules/_export.js:36:28)
```
</details>

**Comment by saket2403** [comment]
@neilpfeiffer Can we get some color updates for help texts on mobile inputs to make it match contrast ratios.

**Comment by bjankord** [comment]
Let's look at grabbing the text color for form fields help text.

**Comment by neilpfeiffer** [comment]
**FIX:** 
Update color here: https://github.com/cerner/terra-framework/blob/1e9427e3312236e517ae8691fd64a6d9d109a0a4/packages/terra-time-input/src/TimeInput.module.scss#L220 
to `rgba(28,31,33,0.63)` to match [Field.module.scss#L59](https://github.com/cerner/terra-core/blob/fabc278bd6e721576e6a558b13f6fbf84d914083/packages/terra-form-field/src/Field.module.scss#L59)

**Comment by neilpfeiffer** [comment]
Will affect both Time-Input and also need to update new screenshots in Date-Time-Picker

**Comment by emilyrohrbough** [comment]
The tests were not updated with this PR change. Looks like the colors have been updated but the test were not updated to remove these disabled accessibility rules:
- https://github.com/cerner/terra-framework/blob/master/packages/terra-date-time-picker/tests/wdio/date-time-picker-mobile-spec.js#L2-L4 
- https://github.com/cerner/terra-framework/blob/master/packages/terra-time-input/tests/wdio/time-input-twelve-hour-mobile-spec.js#L2

**Comment by saket2403** [comment]
On removing axeRules parameter from here:
https://github.com/cerner/terra-framework/blob/cef1c655610e617101faa5fe5baa157c1855c197/packages/terra-date-time-picker/tests/wdio/date-time-picker-mobile-spec.js#L18
I am getting another accessibility issue "duplicate-id-aria"-IDs used in ARIA and labels must be unique.


[Accessibility issue.txt](https://github.com/cerner/terra-framework/files/3334637/Accessibility.issue.txt)

**Comment by emilyrohrbough** [comment]
@saket2403 Can you update the text examples to have unique ids?
